src/utils/word-api.ts

```diff
@@ -61,16 +65,13 @@ class WordApi {
     });
   }
 
-  async setCitations(
+  async updateCitations(
     citations: Array<[number, string, StatefulCitation]>
   ): Promise<unknown> {
     return Word.run(async (context: Word.RequestContext) => {
-      const { contentControls } = context.document;
-      context.load(contentControls, "tag, length");
-      await context.sync();
-      const jabRefCitations = contentControls.items.filter((citation) =>
-        citation.tag.includes("JABREF-CITATION")
-      );
+      const jabRefCitations = (await WordApi.getJabRefCitations(
+        context
+      )) as Array<Word.ContentControl>;
```

Comment:
Why is the `as Array<Word.ContentControl>` assertion needed here? Or asked differently why is `void` needed on the return type of getJabRefCitations?